src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1781,7 +1786,7 @@ private OrganelleDefinition GetOrganelleDefinition(string name)
     }
 
     [DeserializedCallbackAllowed]
-    private void DoOrganellePlaceAction(MicrobeEditorAction action)
+    private void DoOrganellePlaceAction(MicrobeEditorAction action, MicrobeSpecies species)
```

Comment:
I now kinda realize why you added the species parameter to the action callback. But now that I see this...
Does this even make sense to be a member method anymore? If this is given the species to operate on, the data to apply. Then shouldn't this be made into a static method (perhaps even in a new static class for holding all the editor action methods)?